<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualControlDialogV1</class>
 <widget class="QDialog" name="ManualControlDialogV1">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manual System Control</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_15">
     <item>
      <widget class="QGroupBox" name="groupBox_10">
       <property name="title">
        <string>Signal Generation</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Highband Source</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QCheckBox" name="SourceCE">
                 <property name="text">
                  <string>Chip Enable</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="SourceRFEN">
                 <property name="text">
                  <string>RF Enable</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="SourceLocked">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>Locked</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QFormLayout" name="formLayout">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Power:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QComboBox" name="SourceHighPower">
                   <item>
                    <property name="text">
                     <string>-4dbm</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>-1dbm</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>+2dbm</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>+5dbm</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Frequency:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="SIUnitEdit" name="SourceHighFrequency"/>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Lowpass:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QComboBox" name="SourceLowpass">
                   <item>
                    <property name="text">
                     <string>947MHz</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>1880MHz</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>3500MHz</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>None</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string>Lowband Source</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout">
                   <item>
                    <widget class="QCheckBox" name="SourceLowEnable">
                     <property name="text">
                      <string>Enable</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <layout class="QFormLayout" name="formLayout_2">
                     <item row="0" column="0">
                      <widget class="QLabel" name="label_4">
                       <property name="text">
                        <string>Power:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QComboBox" name="SourceLowPower">
                       <item>
                        <property name="text">
                         <string>2mA</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>4mA</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>6mA</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>8mA</string>
                        </property>
                       </item>
                      </widget>
                     </item>
                     <item row="1" column="0">
                      <widget class="QLabel" name="label_5">
                       <property name="text">
                        <string>Frequency:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="1">
                      <widget class="SIUnitEdit" name="SourceLowFrequency"/>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_3">
                <property name="title">
                 <string>Source Switch</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <item>
                  <widget class="QRadioButton" name="SwitchLowband">
                   <property name="text">
                    <string>Lowband</string>
                   </property>
                   <attribute name="buttonGroup">
                    <string notr="true">SourceSwitchGroup</string>
                   </attribute>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="SwitchHighband">
                   <property name="text">
                    <string>Highband</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                   <attribute name="buttonGroup">
                    <string notr="true">SourceSwitchGroup</string>
                   </attribute>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>Attenuator</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QDoubleSpinBox" name="Attenuator">
                 <property name="suffix">
                  <string>db</string>
                 </property>
                 <property name="minimum">
                  <double>-31.750000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>0.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.250000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_5">
              <property name="title">
               <string>Amplifier</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <widget class="QCheckBox" name="AmplifierEnable">
                 <property name="text">
                  <string>Enable</string>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_6">
              <property name="title">
               <string>Port Switch</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QRadioButton" name="Port1Switch">
                 <property name="text">
                  <string>Port 1</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">PortSwitchGroup</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="Port2Switch">
                 <property name="text">
                  <string>Port 2</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">PortSwitchGroup</string>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_11">
       <property name="title">
        <string>Signal Analysis</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox_7">
            <property name="title">
             <string>LO1</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QCheckBox" name="LO1CE">
               <property name="text">
                <string>Chip Enable</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="LO1RFEN">
               <property name="text">
                <string>RF Enable</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="LO1locked">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Locked</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout_3">
               <item row="0" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Freq. Type:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="LO1FreqType">
                 <item>
                  <property name="text">
                   <string>IF1</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Absolute</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Frequency:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="SIUnitEdit" name="LO1Frequency">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>IF1:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="SIUnitEdit" name="IF1"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_8">
            <property name="title">
             <string>LO2</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QCheckBox" name="LO2EN">
               <property name="text">
                <string>Enable</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout_4">
               <item row="0" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Freq. Type:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="LO2FreqType">
                 <item>
                  <property name="text">
                   <string>IF2</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Absolute</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Frequency:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="SIUnitEdit" name="LO2Frequency">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>IF2:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="SIUnitEdit" name="IF2"/>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_9">
            <property name="title">
             <string>Aquisition</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QCheckBox" name="Port1Enable">
               <property name="text">
                <string>Port 1 Enable</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="Port2Enable">
               <property name="text">
                <string>Port 2 Enable</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="RefEnable">
               <property name="text">
                <string>Reference Enable</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout_5">
               <item row="0" column="0">
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>Samples:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="Samples">
                 <property name="minimum">
                  <number>16</number>
                 </property>
                 <property name="maximum">
                  <number>131072</number>
                 </property>
                 <property name="singleStep">
                  <number>16</number>
                 </property>
                 <property name="value">
                  <number>131072</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>Window:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="cbWindow">
                 <item>
                  <property name="text">
                   <string>None</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Kaiser</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Hann</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Flat Top</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_12">
     <property name="title">
      <string>Measurements</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_16">
      <item>
       <widget class="QGroupBox" name="groupBox_16">
        <property name="title">
         <string>Port 1</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_19">
         <item>
          <layout class="QFormLayout" name="formLayout_9">
           <item row="0" column="0">
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>ADC min:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="port1min"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>ADC max:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="port1max"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>Magnitude:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="port1mag"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>Phase:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="port1phase"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Referenced:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="port1referenced"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_14">
        <property name="title">
         <string>Port 2</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_17">
         <item>
          <layout class="QFormLayout" name="formLayout_7">
           <item row="0" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>ADC min:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="port2min"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>ADC max:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="port2max"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Magnitude:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="port2mag"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Phase:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="port2phase"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Referenced:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="port2referenced"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_15">
        <property name="title">
         <string>Reference</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_18">
         <item>
          <layout class="QFormLayout" name="formLayout_8">
           <item row="0" column="0">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>ADC min:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="refmin"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>ADC max:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="refmax"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>Magnitude:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="refmag"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>Phase:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="refphase"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SIUnitEdit</class>
   <extends>QLineEdit</extends>
   <header>CustomWidgets/siunitedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="SourceSwitchGroup"/>
  <buttongroup name="PortSwitchGroup"/>
 </buttongroups>
</ui>
